src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1101,6 +1193,23 @@ DatabaseShardImp::sweep()
     }
 }
 
+Json::Value
+DatabaseShardImp::getDatabaseImportStatus() const
+{
+    Json::Value ret(Json::objectValue);
+
+    if (std::lock_guard lock(mutex_); databaseImportStatus_)
+    {
+        ret[jss::firstShardIndex] = databaseImportStatus_->earliestIndex;
+        ret[jss::lastShardIndex] = databaseImportStatus_->latestIndex;
+        ret[jss::currentShardIndex] = databaseImportStatus_->currentIndex;
```

Comment:
I think it would be cool to include some more information about the current shard import progress, maybe `firstSeq_`, `lastSeq_`, and `acquireInfo_->storedSeqs`.

Reply:
Done